<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="ipLabel">
            <property name="font">
             <font>
              <family>Consolas</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>ip</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="ipLED">
            <property name="text">
             <string extracomment="ip"/>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="portLabel">
            <property name="font">
             <font>
              <family>Consolas</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>port</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="portLED">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="savePBN">
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                              stop: 0 #42a5f5, stop: 1 #1976d2);
    border: 2px solid #1565c0;
    border-radius: 10px;
    color: white;
    font-family: 'Segoe UI', Arial, sans-serif;
    font-size: 10px;
    font-weight: bold;
    padding: 12px 30px;
    margin: 5px;
    text-transform: uppercase;
    letter-spacing: 0.5px;
    
    /* Тень */
    box-shadow: 0 4px 6px rgba(25, 118, 210, 0.3);
}

QPushButton:hover {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                              stop: 0 #2196f3, stop: 1 #0d47a1);
    border: 2px solid #0d47a1;
    box-shadow: 0 6px 8px rgba(25, 118, 210, 0.4);
}

QPushButton:pressed {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                              stop: 0 #1565c0, stop: 1 #0d47a1);
    padding-top: 13px;
    padding-bottom: 11px;
    box-shadow: 0 2px 4px rgba(25, 118, 210, 0.2);
}

QPushButton:disabled {
    background: #bdc3c7;
    border: 2px solid #95a5a6;
    color: #7f8c8d;
    box-shadow: none;
}</string>
          </property>
          <property name="text">
           <string>save</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="loadPBN">
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                              stop: 0 #42a5f5, stop: 1 #1976d2);
    border: 2px solid #1565c0;
    border-radius: 10px;
    color: white;
    font-family: 'Segoe UI', Arial, sans-serif;
    font-size: 10px;
    font-weight: bold;
    padding: 12px 30px;
    margin: 5px;
    text-transform: uppercase;
    letter-spacing: 0.5px;
    
    /* Тень */
    box-shadow: 0 4px 6px rgba(25, 118, 210, 0.3);
}

QPushButton:hover {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                              stop: 0 #2196f3, stop: 1 #0d47a1);
    border: 2px solid #0d47a1;
    box-shadow: 0 6px 8px rgba(25, 118, 210, 0.4);
}

QPushButton:pressed {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                              stop: 0 #1565c0, stop: 1 #0d47a1);
    padding-top: 13px;
    padding-bottom: 11px;
    box-shadow: 0 2px 4px rgba(25, 118, 210, 0.2);
}

QPushButton:disabled {
    background: #bdc3c7;
    border: 2px solid #95a5a6;
    color: #7f8c8d;
    box-shadow: none;
}</string>
          </property>
          <property name="text">
           <string>load</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="sendPBN">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                              stop: 0 #5cd85c, stop: 1 #2ecc71);
    border: 2px solid #27ae60;
    border-radius: 10px;
    color: white;
    font-family: 'Segoe UI', Arial, sans-serif;
    font-size: 10px;
    font-weight: bold;
    padding: 12px 30px;
    margin: 5px;
    text-transform: uppercase;
    letter-spacing: 0.5px;
    
    /* Тень */
    box-shadow: 0 4px 6px rgba(46, 204, 113, 0.3);
}

QPushButton:hover {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                              stop: 0 #52d152, stop: 1 #27ae60);
    border: 2px solid #219653;
    box-shadow: 0 6px 8px rgba(46, 204, 113, 0.4);
}

QPushButton:pressed {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                              stop: 0 #27ae60, stop: 1 #229954);
    padding-top: 13px;
    padding-bottom: 11px;
    box-shadow: 0 2px 4px rgba(46, 204, 113, 0.2);
}

QPushButton:disabled {
    background: #bdc3c7;
    border: 2px solid #95a5a6;
    color: #7f8c8d;
    box-shadow: none;
}</string>
        </property>
        <property name="text">
         <string>send</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="PTE">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>9</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="logTE">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>2</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
